### ISSUE TITLE
default install fails to run due to missing node dependency postcss-discard-comments

### ISSUE THREAD

**Issue by ferrao** [phoenixframework/phoenix#4331]
### Environment

* Elixir version (elixir -v): 1.11.3
* Phoenix version (mix deps): 1.5.9
* Node.js version (node -v): 12.22.1
* NPM version (npm -v): 6.14.12
* Operating system: Arch Linux

### Expected behavior

`mix phx server` should start the web server

### Actual behavior

```
Error: PostCSS plugin postcss-discard-comments requires PostCSS 8.
```

This error is caused due to missing dependency:

```
$ cd  assets
$ npm ls postcss
├─┬ css-loader@3.6.0
│ ├─┬ icss-utils@4.1.1
│ │ └── postcss@7.0.35  deduped
│ ├── postcss@7.0.35 
│ ├─┬ postcss-modules-extract-imports@2.0.0
│ │ └── postcss@7.0.35  deduped
│ ├─┬ postcss-modules-local-by-default@3.0.3
│ │ └── postcss@7.0.35  deduped
│ ├─┬ postcss-modules-scope@2.2.0
│ │ └── postcss@7.0.35  deduped
│ └─┬ postcss-modules-values@3.0.0
│   └── postcss@7.0.35  deduped
└─┬ optimize-css-assets-webpack-plugin@5.0.5
  └── UNMET PEER DEPENDENCY postcss@8.2.15 
```

**Comment by ferrao** [comment]
On `assets/package.json` the following dependency is declared:

```
"optimize-css-assets-webpack-plugin": "^5.0.1",
```

but this ends up installing version `5.0.5`, which satisfies the dependency but introduces the problem.
Forcing 5.0.1 with `npm install optimize-css-assets-webpack-plugin@5.0.1` fixes it.

I guess this module is breaking semver, there is already an [issue](https://github.com/NMFR/optimize-css-assets-webpack-plugin/issues/155) opened on it.

**Comment by jpramassini** [comment]
I just ran into this as well on WSL 2 and Ubuntu with the default install process, thank you @ferrao for the quick fix! 🙌

**Comment by ferrao** [comment]
This issue was fixed in version `5.0.6`, no need to keep this issue opened anymore.
